src/interfaces/wallet.cpp

```diff
@@ -234,6 +234,7 @@ class WalletImpl : public Wallet
         LOCK(m_wallet->cs_wallet);
         return m_wallet->ListLockedCoins(outputs);
     }
+    CAmount getMaxTxFee() override { return g_max_tx_fee; }
```

Comment:
Suggest moving this down below `getDefaultAddressType()` and `getDefaultChangeType()`, since these are other wallet settings the gui asks about.

Reply:
done